src/app/pages/forgot-password/forgot-password.component.scss

```diff
@@ -0,0 +1,16 @@
+@import "typography";
+@import "color";
+
+.header {
+  margin-bottom: 40px;
+
+  .title {
+    margin-bottom: 8px;
+    @include heading-1-medium;
+    color: $basic-1000;
+  }
+}
+
+.resend-button {
+  margin-top: 40px;
```

Comment:
margin-top is bad practice in most cases

Reply:
@dmitry-darly-solutions 
Then what is good practice?

Reply:
@dmitry-darly-solutions 
Fixed.